test/belt/4belt.js

```diff
@@ -0,0 +1,125 @@
+// Utilities
+const Utils = require("../utilities/Utils.js");
+const {
+  impersonates,
+  setupCoreProtocol,
+  depositVault,
+  swapBNBToToken
+} = require("../utilities/hh-utils.js");
+
+const { send } = require("@openzeppelin/test-helpers");
+const BigNumber = require("bignumber.js");
+const IBEP20 = artifacts.require("IBEP20");
+const IDepositor = artifacts.require("IDepositor");
+
+const Strategy = artifacts.require("BeltStrategyMainnet_4Belt");
+
+
+// Vanilla Mocha test. Increased compatibility with tools that integrate Mocha.
+describe("BSC Mainnet 4Belt", function() {
+  let accounts;
+
+  // external contracts
+  let underlying;
+
+  // external setup
+  let belt = "0xE0e514c71282b6f4e823703a39374Cf58dc3eA4f";
+  let eth = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+  let wbnb = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+  let busdAddr = "0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56";
+  let depositorAddr = "0xF6e65B33370Ee6A49eB0dbCaA9f43839C1AC04d5";
+
+  // parties in the protocol
+  let governance;
+  let farmer1;
+
+  // numbers used in tests
+  let farmerBalance;
+
+  // Core protocol contracts
+  let controller;
+  let vault;
+  let strategy;
+
+  async function setupExternalContracts() {
+    underlying = await IBEP20.at("0x9cb73F20164e399958261c289Eb5F9846f4D1404");
+    console.log("Fetching Underlying at: ", underlying.address);
+  }
+
+  async function setupBalance(){
+    busd = await IBEP20.at(busdAddr);
+    liquidityPool = await IDepositor.at(depositorAddr);
+    await swapBNBToToken(farmer1, [wbnb, busdAddr], "100" + "000000000000000000");
+    farmerBusdBalance = await busd.balanceOf(farmer1);
+    await busd.approve(depositorAddr, farmerBusdBalance, {from: farmer1});
+    await liquidityPool.add_liquidity([0, 0, 0, farmerBusdBalance], 0, {from: farmer1});
+    farmerBalance = await underlying.balanceOf(farmer1);
+  }
+
+  before(async function() {
+    governance = "0xf00dD244228F51547f0563e60bCa65a30FBF5f7f";
+    accounts = await web3.eth.getAccounts();
+
+    farmer1 = accounts[1];
+
+    // impersonate accounts
+    await impersonates([governance]);
+
+    let etherGiver = accounts[9];
+    await send.ether(etherGiver, governance, "100" + "000000000000000000")
+
+    await setupExternalContracts();
+    [controller, vault, strategy,,feeForwarder] = await setupCoreProtocol({
```

Comment:
Don't need the feeForwarder here, added the logic to add a path in the hh-utils file. Add `"liquidationPath": [belt, wbnb, eth]` to the inputs instead.

Reply:
resolved it